install.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+if [[ $ASSETS == 'nobuild' ]]; then
+    composer install --no-interaction --no-progress --prefer-dist -o
+else
+    echo "Installing PHP dependencies through Composer..."
+    composer install
```

Comment:
Could you still do `--no-interaction --no-progress --prefer-dist -o` ?
Also, is it this command and user will run to install deps ?